bin/upload.sh

```diff
@@ -14,4 +14,13 @@ if [ "$(ls -A $DIR)" ]; then
     echo "Finished Google Drive upload"
 else
     echo "$DIR is empty"
-fi
\ No newline at end of file
+fi
+
+# uploading mochareporter to Gdrive
+echo "Creating archive of test report $ARCHIVE_REPORTER"
+ARCHIVE_REPORTER=TestReport-$FIXED_BRANCH-$(date +%Y-%m-%d_%H_%M_%S)-$TRAVIS_COMMIT.zip
```

Comment:
Be careful, you try to use a variable before you defined it.

Reply:
Fixed ! 
Thank you .